auth-web/src/views/pay/eft/ShortNameRefundView.vue

```diff
@@ -268,9 +271,9 @@ export default defineComponent({
     onMounted(async () => {
       await loadShortnameDetails(props.shortNameId)
 
-      if (props.eftRefundId) {
+      if (props.shortNameId && props.eftRefundId) {
         state.readOnly = true
-        await loadShortnameRefund(props.eftRefundId)
+        await loadShortnameRefund()
```

Comment:
Shouldn't this be loading for a specific refund Id? There could be multiple refunds on a short name over time